libs/auth-api-lib/src/lib/passkeys-core/models/passkey.model.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  Column,
+  CreatedAt,
+  DataType,
+  Model,
+  Table,
+  UpdatedAt,
+  PrimaryKey,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+
+@Table({
+  tableName: 'passkey',
+})
+export class PasskeyModel extends Model {
```

Comment:
We should use the `InferAttributes` and `InferCreationAttributes`

```suggestion
export class PasskeyModel extends Model<InferAttributes<PasskeyModel>, InferCreationAttributes<PasskeyModel>> {
```

then in the class we can use `CreationAttributes<>` to mark properties which are optional on creation, eg. using the `create` method.